dashboard/test/models/pd/application/teacher_application_test.rb

```diff
@@ -836,6 +840,8 @@ class TeacherApplicationTest < ActiveSupport::TestCase
         assert_equal incomplete, application.reload.principal_approval_state
       end
 
+      refute application.reload.principal_approval_not_required
```

Comment:
Ensuring this doesn't change on create––it was looking like from the Slack thread that it might